services/web/client/source/class/qxapp/data/Permissions.js

```diff
@@ -0,0 +1,122 @@
+/* ************************************************************************
+
+   qxapp - the simcore frontend
+
+   https://osparc.io
+
+   Copyright:
+     2018 IT'IS Foundation, https://itis.swiss
+
+   License:
+     MIT: https://opensource.org/licenses/MIT
+
+   Authors:
+     * Odei Maiz (odeimaiz)
+
+************************************************************************ */
+
+qx.Class.define("qxapp.data.Permissions", {
+  extend: qx.core.Object,
+
+  type : "singleton",
+
+  construct() {
+    this.addAction("tester", "test");
+  },
+
+  events: {
+    "UserProfileRecieved": "qx.event.type.Event"
```

Comment:
There is no convention for events naming? I have read ``getSuccess`` as well

Reply:
Nope afaik. But we could adopt the official camelCase